src/Microsoft.Health.Fhir.Liquid.Converter.FunctionalTests/FunctionalTests.cs

```diff
@@ -22,30 +23,44 @@ public class FunctionalTests
     {
         public static IEnumerable<object[]> GetDataForHl7v2()
         {
-            var data = new List<object[]>
+            var data = new List<string[]>
             {
-                new object[] { @"ADT_A01", @"ADT01-23.hl7", @"ADT01-23-expected.json" },
-                new object[] { @"ADT_A01", @"ADT01-28.hl7", @"ADT01-28-expected.json" },
-                new object[] { @"ADT_A01", @"ADT04-23.hl7", @"ADT04-23-expected.json" },
-                new object[] { @"ADT_A01", @"ADT04-251.hl7", @"ADT04-251-expected.json" },
-                new object[] { @"ADT_A01", @"ADT04-28.hl7", @"ADT04-28-expected.json" },
-                new object[] { @"OML_O21", @"MDHHS-OML-O21-1.hl7", @"MDHHS-OML-O21-1-expected.json" },
-                new object[] { @"OML_O21", @"MDHHS-OML-O21-2.hl7", @"MDHHS-OML-O21-2-expected.json" },
-                new object[] { @"ORU_R01", @"LAB-ORU-1.hl7", @"LAB-ORU-1-expected.json" },
-                new object[] { @"ORU_R01", @"LAB-ORU-2.hl7", @"LAB-ORU-2-expected.json" },
-                new object[] { @"ORU_R01", @"LRI_2.0-NG_CBC_Typ_Message.hl7", @"LRI_2.0-NG_CBC_Typ_Message-expected.json" },
-                new object[] { @"ORU_R01", @"ORU-R01-RMGEAD.hl7", @"ORU-R01-RMGEAD-expected.json" },
-                new object[] { @"VXU_V04", @"IZ_1_1.1_Admin_Child_Max_Message.hl7", @"IZ_1_1.1_Admin_Child_Max_Message-expected.json" },
-                new object[] { @"VXU_V04", @"VXU.hl7", @"VXU-expected.json" },
+                new[] { @"ADT_A01", @"ADT01-23.hl7", @"ADT01-23-expected.json" },
+                new[] { @"ADT_A01", @"ADT01-28.hl7", @"ADT01-28-expected.json" },
+                new[] { @"ADT_A01", @"ADT04-23.hl7", @"ADT04-23-expected.json" },
+                new[] { @"ADT_A01", @"ADT04-251.hl7", @"ADT04-251-expected.json" },
+                new[] { @"ADT_A01", @"ADT04-28.hl7", @"ADT04-28-expected.json" },
+                new[] { @"OML_O21", @"MDHHS-OML-O21-1.hl7", @"MDHHS-OML-O21-1-expected.json" },
+                new[] { @"OML_O21", @"MDHHS-OML-O21-2.hl7", @"MDHHS-OML-O21-2-expected.json" },
+                new[] { @"ORU_R01", @"LAB-ORU-1.hl7", @"LAB-ORU-1-expected.json" },
+                new[] { @"ORU_R01", @"LAB-ORU-2.hl7", @"LAB-ORU-2-expected.json" },
+                new[] { @"ORU_R01", @"LRI_2.0-NG_CBC_Typ_Message.hl7", @"LRI_2.0-NG_CBC_Typ_Message-expected.json" },
+                new[] { @"ORU_R01", @"ORU-R01-RMGEAD.hl7", @"ORU-R01-RMGEAD-expected.json" },
+                new[] { @"VXU_V04", @"IZ_1_1.1_Admin_Child_Max_Message.hl7", @"IZ_1_1.1_Admin_Child_Max_Message-expected.json" },
+                new[] { @"VXU_V04", @"VXU.hl7", @"VXU-expected.json" },
             };
-            return data.Select(item => new object[]
+            return data.Select(item => new[]
             {
                 Convert.ToString(item[0]),
```

Comment:
Seems we don't need ToString here?

Reply:
Yes, this is redundant now and is removed in the update.